requirements_dev.txt

```diff
@@ -3,81 +3,57 @@ hypothesis
 pyfakefs
 hypothesis-fspaths
 coverage
+
 Pillow
 pandas
-matplotlib!=3.5.2
-soundfile
-boto3
-google-cloud-storage
-google-cloud-aiplatform
-kubernetes
 moviepy
 imageio
+matplotlib!=3.5.2
+soundfile
+rdkit-pypi; sys_platform != 'darwin' or (sys_platform == 'darwin' and platform.machine != 'arm64')
+rdkit-pypi; python_version > '3.7' and sys_platform == 'darwin' and platform.machine == 'arm64'
+plotly
+bokeh
+
+tqdm
+
 ipython
 ipython<8.13; python_version < '3.9'
 ipykernel
 nbclient
+
 scikit-learn
 tensorflow>=1.15.2; sys_platform != 'darwin'
 tensorflow>=1.15.2; python_version > '3.6' and sys_platform == 'darwin' and platform.machine != 'arm64'
 tensorflow-macos; python_version > '3.7' and python_version < '3.11' and sys_platform == 'darwin' and platform.machine == 'arm64'
+tensorboard
 torch
 torchvision
-plotly
-bokeh
-tqdm
-docker
-tensorboard
 jax[cpu]; sys_platform == 'darwin' or sys_platform == 'linux'
-fastcore; python_version > '3.6'
-fastcore==1.3.29; python_version == '3.6'
+
+fastcore
 pyarrow
 metaflow>=2.3.5
 xgboost
 lightgbm
-rdkit-pypi; sys_platform != 'darwin' or (sys_platform == 'darwin' and platform.machine != 'arm64')
-rdkit-pypi; python_version > '3.7' and sys_platform == 'darwin' and platform.machine == 'arm64'
-httpx>=0.22.0  # 0.23.0 dropped Python 3.6; we can upgrade once we drop it too
-respx>=0.20.0  # 0.21.0 dropped Python 3.6; we can upgrade once we drop it too
-pytest-benchmark
+mlflow
+kfp<2.0.0
+urllib3<2
```

Comment:
had to pin it... otherwise pip is unable to resolve all the dependencies here. 
Future solution is to break down this huge requirements file!